installer.nsh

```diff
@@ -104,6 +62,11 @@
         ${EndIf}
         FileWrite $installLog "- $REMOTE_CLI_EXE -j service install     [$0]  $1$\r$\n"
 
+        CopyFiles $REMOTE_CLI_EXE $PATH_REMOTE_DIR
+        CopyFiles $MUXER $PATH_REMOTE_DIR
+        CopyFiles $DEMUXER $PATH_REMOTE_DIR
+        CopyFiles $CONNECTD $PATH_REMOTE_DIR
+
```

Comment:
Want to set path var instead of copying.

https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7

Reply:
See above link. No copying.